src/data/nat/basic.lean

```diff
@@ -606,6 +606,16 @@ by rw [←nat.mod_add_div a c, ←nat.mod_add_div b c, ←h, ←nat.sub_sub, nat
 lemma dvd_sub_mod (k : ℕ) : n ∣ (k - (k % n)) :=
 ⟨k / n, nat.sub_eq_of_eq_add (nat.mod_add_div k n).symm⟩
 
+lemma add_mod (a b n : ℕ) : (a + b) % n = ((a % n) + (b % n)) % n :=
+begin
+  conv
+  begin
+    to_lhs,
+    rw [←mod_add_div a n, ←mod_add_div b n, ←add_assoc, add_mul_mod_self_left,
+        add_assoc, add_comm _ (b % n), ←add_assoc, add_mul_mod_self_left]
+  end
```

Comment:
```suggestion
  conv_lhs {
    rw [←mod_add_div a n, ←mod_add_div b n, ←add_assoc, add_mul_mod_self_left,
        add_assoc, add_comm _ (b % n), ←add_assoc, add_mul_mod_self_left] }
```
Stylistic thingy: We usually don't nest `begin ... end`-blocks but use `{ ... }` instead. Also, there's a little shortcut `conv_lhs`.

Reply:
I'd like to suggest that both these things should be covered in docs/extras/conv.md.